packages/runtime/container-runtime/src/test/dataStoreContext.spec.ts

```diff
@@ -437,12 +441,15 @@ describe("Data Store Context Tests", () => {
 				const expectedEvents = [
 					{
 						eventName: "FluidDataStoreContext:DataStoreMessageSubmittedInSummarizer",
-						packageName: packagePathToTelemetryProperty(packageName),
+						type: DataStoreMessageType.ChannelOp,
 						fluidDataStoreId: {
+							tag: TelemetryDataTag.CodeArtifact,
 							value: dataStoreId,
+						},
+						fullPackageName: {
```

Comment:
Why rename this property?

Reply:
it is inconsistently used throughout the file. this value broke the fewest tests